### DISCUSSION TITLE
Can mod_auth_openidc run on solaris11 systems?

### DISCUSSION THREAD

**Discussion by 2168028160** [OpenIDC/mod_auth_openidc#510] [Q&A] [answered]
We encountered the following problem when installing mod_auth_openidc on a solaris 11.4 system.
If solaris is supported, how should I solve the following problems. Thanks.

1. with make.
   make: Fatal error in reader: Makefile, line 18: Unexpected end of line seen

2. with gmake.
   /usr/bin/apxs  -Wc,"-DNAMEVER=\"mod_auth_openidc-2.4.5\" -I/usr/include/jansson   -I/usr/local/include -I/usr/include/jansson   -I/usr/include/pcre   -DUSE_MEMCACHE " -Wl,"-lssl -lcrypto   -lcurl   -ljansson   -L/usr/local/lib -lcjose -ljansson -lcrypto   -lpcre    " -Wc,-Wall -Wc,-g -c src/mod_auth_openidc.c src/cache/file.c src/cache/shm.c src/cache/common.c src/oauth.c src/proto.c src/config.c src/util.c src/authz.c src/session.c src/metadata.c src/jose.c src/parse.c src/pcre_subst.c src/cache/memcache.c
/usr/apr/1.5/build/sparcv9/libtool --silent --mode=compile cc -prefer-pic -m64 -xO4 -xtarget=ultra2 -xarch=sparcvis -xchip=ultra2 -Qoption cg -xregs=no%appl -W2,-xwrap_int -xmemalign=16s -mt  -DSOLARIS2=11 -D_POSIX_PTHREAD_SEMANTICS -D_REENTRANT -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64  -I/usr/apache2/2.4/include  -I/usr/apr/1.5/include   -I/usr/apr-util/1.5/include -I/usr/include/openldap -DNAMEVER="mod_auth_openidc-2.4.5" -I/usr/include/jansson   -I/usr/local/include -I/usr/include/jansson   -I/usr/include/pcre   -DUSE_MEMCACHE  -Wall -g  -c -o src/mod_auth_openidc.lo src/mod_auth_openidc.c && touch src/mod_auth_openidc.slo
/usr/apr/1.5/build/sparcv9/libtool: line 1115: cc: command not found
apxs:Error: Command failed with rc=65536
.
gmake: *** [src/mod_auth_openidc.la] Error 1

**Comment by zandbelt** [comment, marked as answer]
Solaris is only supported under a commercial support subscription

**Reply by 2168028160** [reply]
Is it possible to provide a trial version of solaris11.3?
